config/admin.ts

```diff
@@ -1,22 +1,27 @@
 export default ({ env }) => ({
   auth: {
-    secret: env('ADMIN_JWT_SECRET'),
+    secret: env("ADMIN_JWT_SECRET"),
```

Comment:
Prettier makes you pick either single or double quotes, and I like double quotes, so here we are ☺️